homeassistant/components/velux/__init__.py

```diff
@@ -73,6 +75,8 @@ async def async_reboot_gateway(service_call: ServiceCall) -> None:
         host = self._domain_config.get(CONF_HOST)
         password = self._domain_config.get(CONF_PASSWORD)
         self.pyvlx = PyVLX(host=host, password=password)
+        # the gateway cannot fetch more than one time in parallel
+        self.semaphore_poll_data = asyncio.Semaphore(1)
```

Comment:
Is this a limitation only when calling `get_limitation`, or is is it a limitation also for other pyvlx calls?
Why does Home Assistant need to serialize the calls, shouldn't that be done by the library?

Reply:
It is rather a safeguards, i have test it local with 6 windows mostly (90%)there were no issues but sometimes the Gateway timed out. 
After setting a semaphore I got not issues.
I don't know what is the best practice for this behavior.
Almost data from the Velux comes as an event. Only the limitation sensor is read out by polling. This is a behavior of the gateway.
I have never seen issues with other pyvlx calls.

I think the semaphore implementation in pyvlx is harder than in home assistant.

Reply:
> I think the semaphore implementation in pyvlx is harder than in home assistant.

Why is that?

Reply:
I test this behaviour again and i have no issues with parallel call. Semaphore is removed. https://github.com/home-assistant/core/pull/64628/commits/9be35c5823aa360bb284bb403fe62be658293944